cpp/src/dbscan/dbscan.cuh

```diff
@@ -65,7 +67,7 @@ size_t compute_batch_size(size_t& estimated_memory,
 
   // To avoid overflow, we need: batch_size <= MAX_LABEL / n_rows (floor div)
   Index_ MAX_LABEL = std::numeric_limits<Index_>::max();
-  if (batch_size > MAX_LABEL / n_rows) {
+  if (batch_size > static_cast<std::size_t>(MAX_LABEL / n_rows)) {
```

Comment:
I'll just mention it for this file, although we need to check it for everywhere that we're using `std::size_t`: don't forget to `#include <cstddef>`

Reply:
Done throughout the library.